cloud-config/deploy-run-scripts/ami-java.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Oracle Java 11 installation
+echo -e "\n$ENCD_INSTALL_TAG $(basename $0)"
+
+# Check previous failure flag
+if [ -f "$encd_failed_flag" ]; then
+    echo -e "\n\t$ENCD_INSTALL_TAG $(basename $0) Skipping: encd_failed_flag exits"
+    exit 1
+fi
+echo -e "\n\t$ENCD_INSTALL_TAG $(basename $0) Running"
+
+# Script Below
+# Manually install java
+sudo -u ubuntu aws s3 cp --region=us-west-2 --recursive s3://encoded-conf-prod/encd-tars ~ubuntu/encd-tars
```

Comment:
this is just java?

Reply:
Yea, we download the tar from s3...At least now it only downloads when building the base ami instead of ever time we create a demo.